dns/records/resources.tf

```diff
@@ -8,6 +8,7 @@ locals {
         zone_name           = zone_name
         resource_group_name = zone_cfg["resource_group_name"]
         target              = record_cfg["target"]
+        ttl                 = record_cfg["ttl"]
```

Comment:
Could we set a default TTL here? Using `try` maybe

Reply:
changed to try(record_cfg["ttl"], 300)
tested ok